src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -342,6 +344,27 @@ public ExamResource(UserRepository userRepository, CourseRepository courseReposi
         return ResponseEntity.ok(exam);
     }
 
+    /**
+     * GET /courses/:courseId/exams/:examId/stats-for-exam-assessment-dashboard A collection of useful statistics for the tutor course dashboard, including: - number of submissions to the course - number of
+     * assessments - number of assessments assessed by the tutor - number of complaints
```

Comment:
This could be formatted as list with new lines.

Reply:
i put some line breaks in.